README.md

```diff
@@ -504,7 +504,7 @@ Recommended Client-side Libraries
 - [X-editable](http://vitalets.github.io/x-editable/) - Edit form elements inline.
 - [Offline.js](http://github.hubspot.com/offline/docs/welcome/) - Detect when user's internet connection goes offline.
 - [Alertify.js](http://fabien-d.github.io/alertify.js/) - Sweet looking alerts and browser dialogs.
-- [selectize.js](http://brianreavis.github.io/selectize.js/) - Styleable select elements and input tags.
+- [selectize.js](https://github.com/selectize/selectize.js/) - Styleable select elements and input tags.
```

Comment:
http://selectize.github.io/selectize.js might be a better link.

Reply:
Yes, would be better to have landing page link instead of repository link for consumption purpose